src/pipeline_config.py

```diff
@@ -16,6 +16,11 @@
 MEAN = [0.485, 0.456, 0.406]
 STD = [0.229, 0.224, 0.225]
 
+# ASPECT_RATIOS = [1/2., 1/1., 2/1.]
+# SCALE_RATIOS = [1., pow(2,1/3.), pow(2,2/3.)]
+ASPECT_RATIOS = [1 / 1.]
```

Comment:
@taraspiotr I think that if we put aspect ratios here we should have it in neptune.yaml as well. It is part of the architucture. One can simply pass it as string and run eval here. For reference check open-solution to home-credit

Reply:
@jakubczakon ok